CHANGE-NOTES.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+## v3.0.3
+- Enabled Ctrl + mouse click to work as a right-click for Mac users
```

Comment:
also for consistency we refer mac with `macOS`

> No, they are essentially the same thing — just named differently. In fact, three terms were used at different times with reference to Apple’s operating system: Mac OS X, OS X, and macOS. Mac OS X was the official naming through version 10.7, from 2001 to 2011. In the next four years, the OS X names were used. Finally, Apple shifted to "macOS" with the release of macOS High Sierra in 2016. The latter helped standardize the naming of Apple’s operating systems — macOS, iOS, tvOS, iPadOS, etc. 

~ https://setapp.com/how-to/full-list-of-all-macos-versions

let's change it here and within CONTRIBUTING.md#Plugin signing as part of the CI publish process through the Gradle Plugin

Reply:
Done as suggested in https://github.com/VirtusLab/git-machete-intellij-plugin/pull/1197